plugins/jobs/src/js/components/JobsOverviewTable.js

```diff
@@ -130,14 +133,16 @@ export default class JobsOverviewTable extends React.Component {
 
         // to avoid duplicates in our listing, we need to hack this… (again, this whole thing needs refactoring…)
         // we're getting an array back with the wrapping Object.values(…)
-        acc[isGroup ? `path:${id}` : `job:${id}`] = {
-          id,
-          isGroup,
-          name,
-          schedules,
-          status,
-          lastRun
-        };
+        if (!isGroup || acc[`path:${id}`] === undefined) {
```

Comment:
This is not needed to fix the bug, but it's just a way to avoid adding the same group twice if it contains multiple jobs.